.github/workflows/container.yml

```diff
@@ -0,0 +1,36 @@
+name: Container build
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'container-images/**'
+      - '.github/workflows/container.yml'
+  pull_request:
+    paths:
+      - 'container-images/**'
+      - '.github/workflows/container.yml'
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  container:
+    name: ${{ matrix.container }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - candlepin
+          - foreman
+          - foreman-proxy
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build ${{ matrix.container }} container
+        run: make -C container-images/${{ matrix.container }} build VERSION=ci
```

Comment:
`VERSION=ci` is only needed because the variable would be otherwise unset for Candlepin:
https://github.com/theforeman/foreman-quadlet/blob/2df4338f70f9bc1903fc6becb077a21034c82638/container-images/candlepin/Makefile#L2
It's otherwise unused (the `Containerfile` has a hard-coded 4.4.14 in it right now), so I wonder if we should instead fix up the `Makefile`?

Reply:
> `VERSION=ci` is only needed because the variable would be otherwise unset for Candlepin:
> 
> https://github.com/theforeman/foreman-quadlet/blob/2df4338f70f9bc1903fc6becb077a21034c82638/container-images/candlepin/Makefile#L2
> 
> 
> It's otherwise unused (the `Containerfile` has a hard-coded 4.4.14 in it right now), so I wonder if we should instead fix up the `Makefile`?

Makes sense to me to keep this the same across each build.

Reply:
"across each Makefile" you mean? ack

Reply:
yep

Reply:
done